docs/faq.md

```diff
@@ -20,15 +20,42 @@ Skin can be simulated using the [AdnSkin](maya/skin) deformer. This deformer req
 
 Similarly to a skin simulation setup, the fascia can be simulated using [AdnSkin](maya/skin) deformer as well. In this case, it is recommended to use values of *Rest Length Multiplier* lower than 1. This deformer also requires one mesh (a single target, i.e. the muscle geometries combined) or multiple meshes (multiple targets, i.e. the muscle geometries separated) to drive the simulation and a mesh to apply the deformer to (fascia geometry).
 
-## Can I simulate muscles, fascia and skin all coupled?
+## How can I generate the fascia geometry?
 
-Yes, you can simulate muscles, fascia and skin following these steps:
+In order to obtain a plausible fascia geometry, we can use two techniques that require the muscle geometries and the skin geometry as inputs.
+
+The easiest option is to use the shrinkwrap deformer provided by Maya. This solution works better for low poly geometries.
+
+- Create a copy of the skin geometry that will become your fascia.
+- Select the combined geometry of all muscles, then select the copy of the skin geometry.
+- Apply the shrinkwrap deformer and adjust its settings until the desired fascia geometry is achieved.
+- Delete the history of the fascia geometry.
+
+For high-end fascia simulations, a better option would be to create the fascia geometry using a pressured cloth simulation. nCloth can be used to achieve this:
+
+- Create a copy of the skin geometry that will become your fascia.
+- Select the fascia geometry and click nCloth > Create nCloth.
+- Select the muscle geometries and click on nCloth > Create Passive Collider.
+- Select the nucleus node and set the gravity to 0.
+- Select the nCloth node and keyframe the pressure attribute to 0 at the first frame of the scene.
+- Advance some frames (e.g. ten frames) and keyframe the pressure to -1.
+- Select the fascia geometry and click nCloth > Create New nCache > nObject.
+- Once the simulation finishes, choose the fascia that looks best within the simulated frame range.
+
+## How can I simulate fat?
+
+Fat can be simulated using the [AdnFat](maya/fat) deformer. This deformer requires one base mesh (i.e. the fascia) to drive the fat simulation and a mesh to apply the deformer to (fat mesh). Both meshes must have the same vertex count and triangulation. A simple setup is explained [here](maya/simple_setup#adnfat).
+
+## Can I simulate muscles, fascia, fat and skin all coupled?
+
+Yes, you can simulate muscles, fascia, fat and skin following these steps:
 
 - Configure every muscle geometry with an AdnMuscle deformer.
 - Combine all muscle geometries into a single geometry.
-- Shrinkwrap the skin geometry to the combined muscles geometry to obtain the fascia geometry.
+- Shrinkwrap the skin geometry to the combined muscles geometry to obtain the fascia geometry. Alternatively, pressure can be simulated on the skin geometry so that it is adjusted to the muscle geometries to obtain the fascia geometry.
```

Comment:
I suggest:

```suggestion
- Generate the fascia geometry. Check this answer to know how to complete this step.
```

And make "this answer" a link to the question above.

Important!! The step "Combine all muscle geometries into a single geometry" has to be removed. This is not a requirement anymore from version v1.2.0 actually

Reply:
It still is a requirement for the shrinkwrap though, not for setting up the fascia. We should differentiate. For the step below "Configure the fascia with an AdnSkin deformer: Select the combined muscles geometry," and some other steps we should remove the combined word.

Reply:
Removed the step "Combine all muscle geometries..." from this question. Also linked the step "Generate the fascia geometry" with its corresponding question.